# Transplacental metastasis

> this repository contains code base for bulk RNA-seq anlisis used in research about genes and pathways assosiated with Transplacental metastasis

# Referenses

## Data

1. **RNA-seq** We used bulk RNA-seq expirement desined for Determining sex-specific gene expression differences in human chorion trophoblast cells (BioProject - `PRJNA1215956`) by College of Medicine, University of the Philippines Manila. For our reserch only control samples were selected.
2. **Reference transcriptome and Transcript-to-gene map** We used [kallisto-transcriptome-indices v1](https://github.com/pachterlab/kallisto-transcriptome-indices/releases/tag/v1) `human_index_standard` set as reference transcriptome for kallisto quantification along with corresponding transcript-to-gene mapping file
3. **Genes Signatures** We used [C6: oncogenic signature gene sets](https://www.gsea-msigdb.org/gsea/msigdb/human/collections.jsp#C6) by MSigDB `v2024.1.Hs_GMTs` providing functional oncogenic associated signatures for gene sets

## Software

1. Babraham Bioinformatics - FastQC A Quality Control tool for High Throughput Sequence Data [Электронный ресурс]. URL: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/ (дата обращения: 23.02.2025).
2. Di Tommaso P. [и др.]. Nextflow enables reproducible computational workflows // Nature Biotechnology. 2017. № 4 (35). C. 316–319.
3. Ewels P. [и др.]. MultiQC: summarize analysis results for multiple tools and samples in a single report // Bioinformatics (Oxford, England). 2016. № 19 (32). C. 3047–3048.
4. Fang Z., Liu X., Peltz G. GSEApy: a comprehensive package for performing gene set enrichment analysis in Python // Bioinformatics. 2023. № 1 (39). C. btac757.
5. GitHub - ncbi/sra-tools: SRA Tools [Электронный ресурс]. URL: https://github.com/ncbi/sra-tools (дата обращения: 23.02.2025).
6. Harris C. R. [и др.]. Array programming with NumPy // Nature. 2020. № 7825 (585). C. 357–362.
7. Love M. I., Huber W., Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2 // Genome Biology. 2014. № 12 (15). C. 1–21.
8. Nextflow enables reproducible computational workflows | Nature Biotechnology [Электронный ресурс]. URL: https://www.nature.com/articles/nbt.3820 (дата обращения: 23.02.2025).
9. Near-optimal probabilistic RNA-seq quantification | Nature Biotechnology [Электронный ресурс]. URL: https://www.nature.com/articles/nbt.3519 (дата обращения: 23.02.2025).
10. org.Hs.eg.db // Bioconductor [Электронный ресурс]. URL: http://bioconductor.org/packages/org.Hs.eg.db/ (дата обращения: 23.02.2025).
11. Soneson C., Love M. I., Robinson M. D. Differential analyses for RNA-seq: transcript-level estimates improve gene-level inferences // F1000Research. 2015. (4). C. 1521.
12. team T. pandas development pandas-dev/pandas: Pandas // 2024.
13. Venables W. N., Ripley B. D. Modern applied statistics with S / W. N. Venables, B. D. Ripley, 4th ed-е изд., New York: Springer, 2002.
14. Waskom M. seaborn: statistical data visualization // Journal of Open Source Software. 2021. № 60 (6). C. 3021.
15. Wickham H. ggplot2: Elegant Graphics for Data Analysis / H. Wickham, 2nd ed. 2016-е изд., Cham: Springer International Publishing : Imprint: Springer, 2016. 1 c.
